packages/back-end/src/models/FeatureModel.ts

```diff
@@ -640,34 +640,48 @@ export async function addFeatureRule(
   revision: FeatureRevisionInterface,
   env: string,
   rule: FeatureRule,
-  user: EventAuditUser
+  user: EventAuditUser,
+  resetReview: boolean
 ) {
   if (!rule.id) {
     rule.id = generateRuleId();
   }
 
   const changes = {
     rules: revision.rules || {},
+    status: revision.status,
   };
   changes.rules[env] = changes.rules[env] || [];
   changes.rules[env].push(rule);
-
-  await updateRevision(revision, changes, {
-    user,
-    action: "add rule",
-    subject: `to ${env}`,
-    value: JSON.stringify(rule),
-  });
+  if (
+    (revision.status === "approved" ||
+      revision.status === "changes-requested") &&
+    resetReview
+  ) {
+    changes.status = "pending-review";
+  }
```

Comment:
Made a comment in FeatureRevisionModel too, but we can remove this logic here and have `updateRevision` determine this internally.

Reply:
same here just forgot to remove